dts/arm/st/h7/stm32h735.dtsi

```diff
@@ -49,6 +49,66 @@
 			health-test-magic = <0x17590abc>;
 			health-test-config = <0xaa74>;
 		};
+
+		lptim1: timers@40002400 {
+			compatible = "st,stm32-lptim";
+			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00000200>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40002400 0x400>;
+			interrupts = <93 1>;
+			interrupt-names = "wakeup";
+			status = "disabled";
+			label = "LPTIM_1";
+		};
+
+		lptim2: timers@58002400 {
+			compatible = "st,stm32-lptim";
+			clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00000200>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x58002400 0x400>;
+			interrupts = <138 1>;
+			interrupt-names = "wakeup";
+			status = "disabled";
+			label = "LPTIM_2";
+		};
+
+		lptim3: timers@58002800 {
+			compatible = "st,stm32-lptim";
+			clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00000400>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x58002800 0x400>;
+			interrupts = <139 1>;
+			interrupt-names = "wakeup";
+			status = "disabled";
+			label = "LPTIM_3";
+		};
+
+		lptim4: timers@58002c00 {
+			compatible = "st,stm32-lptim";
+			clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00000800>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x58002c00 0x400>;
+			interrupts = <140 1>;
+			interrupt-names = "wakeup";
+			status = "disabled";
+			label = "LPTIM_4";
+		};
+
+		lptim5: timers@58003000 {
+			compatible = "st,stm32-lptim";
+			clocks = <&rcc STM32_CLOCK_BUS_APB4 0x00001000>;
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x58003000 0x400>;
+			interrupts = <141 1>;
+			interrupt-names = "wakeup";
+			status = "disabled";
+			label = "LPTIM_5";
```

Comment:
LPTIM driver implementation doesn't support multiples instances today.
Let's remove instances 2 to 5 for now in order to avoid confusion.

Reply:
done